templates/document_head.hbs

```diff
@@ -5,13 +5,14 @@
 <script type="importmap">
     {
         "imports": {
-            "vendor": "{{asset "vendor.js"}}",
-            "shared": "{{asset "shared.js"}}"
+            "vendor": "{{asset 'vendor.js'}}",
+            "shared": "{{asset 'shared.js'}}",
+            "new-request-form": "{{asset 'NewRequestForm.js'}}"
```

Comment:
Good point! Makes sense to align 👍🏼 
In regards to the asset name, since it's a bundle and we don't have a default export, would it make more sense to name it `new-request-form.js` or `new_request_form.js`?